### PR TITLE
mvebu: add LED support for GL.iNet GL-MV1000

### PR DESCRIPTION
**Pull request by ghost**
This patch enables the leds for the GL.iNet GL-MV1000 / Brume

### CODE REVIEW

File: target/linux/mvebu/files/arch/arm64/boot/dts/marvell/armada-3720-gl-mv1000.dts [hunk 1]
```diff
@@ -29,6 +29,28 @@
 			3300000 0x0>;
 		enable-active-high;
 	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led-1 {
+			label = "gl-mv1000:green:vpn";
```
Comment (adschm) [inline target/linux/mvebu/files/arch/arm64/boot/dts/marvell/armada-3720-gl-mv1000.dts:37]: If you add this, please use the recently preferred syntax without the useless model, i.e. "green:vpn" etc.

File: target/linux/mvebu/files/arch/arm64/boot/dts/marvell/armada-3720-gl-mv1000.dts [hunk 2]
```diff
@@ -29,6 +29,28 @@
 			3300000 0x0>;
 		enable-active-high;
 	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led-1 {
```
Comment (adschm) [inline target/linux/mvebu/files/arch/arm64/boot/dts/marvell/armada-3720-gl-mv1000.dts:36]: consider naming this after the label:
vpn_green etc.

File: target/linux/mvebu/files/arch/arm64/boot/dts/marvell/armada-3720-gl-mv1000.dts [hunk 3]
```diff
@@ -29,6 +29,28 @@
...
+			label = "gl-mv1000:green:vpn";
+			gpios = <&gpionb 11 GPIO_ACTIVE_LOW>;
+			default-state = "off";
```
Comment (adschm) [inline target/linux/mvebu/files/arch/arm64/boot/dts/marvell/armada-3720-gl-mv1000.dts:39]: default-state=off is default and can be dropped.

File: target/linux/mvebu/cortexa53/base-files/etc/board.d/01_leds [hunk 4]
```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+. /lib/functions/uci-defaults.sh
+
+board=$(board_name)
+boardname="${board##*,}"
```
Comment (adschm) [inline target/linux/mvebu/cortexa53/base-files/etc/board.d/01_leds:6]: boardname variable can be dropped.

**Comment by 1Jeff1** [comment]
@adschm I made the changes  requested.

**Comment by adschm** [comment]
Just saw that your name in Signed-off is not properly capitalized.

**Comment by 1Jeff1** [comment]
> Just saw that your name in Signed-off is not properly capitalized.

OK is it fixed now.

**Comment by adschm** [comment]
I've added aliases as well and will merge this later today.

**Comment by mtis88** [comment]
Any change this can be included in 21.02?

**Comment by adschm** [comment]
> Any chance this can be included in 21.02?

Done.

**Pull request closed without merging**